tests/admin/clickhouse_migrations/test_api.py

```diff
@@ -100,14 +110,24 @@ def test_run_reverse_migrations(admin_api: FlaskClient, action: str) -> None:
     method = "run_migration" if action == "run" else "reverse_migration"
 
     with patch(
-        "snuba.settings.ADMIN_ALLOWED_MIGRATION_GROUPS", {"system", "invalid_group"}
+        "snuba.settings.ADMIN_ALLOWED_MIGRATION_GROUPS",
+        {
+            "system": "AllMigrationsPolicy",
+            "invalid_but_allowed_group": "AllMigrationsPolicy",
+            "generic_metrics": "NoMigrationsPolicy",
+            "events": "NonBlockingMigrationsPolicy",
+        },
     ):
         # invalid action
-        response = admin_api.post("/migrations/system/invalid_action/0001_migrations/")
+        response = admin_api.post(
+            "/migrations/system/invalid_but_allowed_group/0001_migrations/"
```

Comment:
To be honest, I'm not a big fan of this URL scheme. I don't understand what `system` means here, and would personally prefer something that aligns more closely with our CLI commands, for example:

`POST /migrations/<action> -d {"group": "some_group", "migration_id": "0001...", }` where action is one of run, reverse, list, etc

Reply:
I think it makes sense to have the group in the top level since we also have a list for the groups

what if we had

`/migrations_groups/<group>/list`
`/migrations_groups/<group>/<run | reverse>/` -> and this can have the migration id, fake, force and dry run in the data?

Reply:
Wouldn't there be cases though where passing a group might not be necessary? e.g. we might want the ability to list everything at once or get all the groups. How would those fit in?

IMO it's cleaner to put all migration system functionality under `/migrations` rather than potentially split the interface in two,

Reply:
also fixed a typo on this line. it should be `"/migrations/system/invalid_action/0001_migrations/"`